lib/mock/mock-pool.js

```diff
@@ -54,6 +54,10 @@ class MockPool extends Pool {
     )
   }
 
+  cleanMocks () {
+    this[kDispatches].length = 0
```

Comment:
```suggestion
    this[kDispatches] = []
```

I think this is more deliberate